examples/proc_GDS_capture_int.py

```diff
@@ -1,3 +1,4 @@
+# PR docstring needed
```

Comment:
Sorry -- I missed this before -- you just need a description.

Reply:
Is this the processing for your "simple capture"? Or what is this?

Reply:
yes processing for individual capture of single pulse length - mentioned acq file in docstring